entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+source $NVM_DIR/nvm.sh
+nvm use ${NODE_VERSION}
+
+if [[ "${BASE_DIR}" == "" ]]; then
+    BASE_DIR=.
+fi
+
+export DEBUG_LEVEL=3
+export DEBUG="exachimp:*"
+export ROOT_URL="http://127.0.0.1:3001"
+
+if [[ "$1" == "test" ]]; then
+    # xvfb-run -s "-ac -screen 0 1920x1080x24" \
```

Comment:
Remove commented out line?

Reply:
I think the webdriverio cmd line args are sufficient to have the tooling internally launch xvfb but I'm not 100% convinced yet. I added more description around it and will remove it once https://exabyte.atlassian.net/browse/SOF-5465 is completed.